src/components/LoadingSkeletons/LoadingSkeleton.tsx

```diff
@@ -9,8 +9,6 @@ import Panel from '../Panel/Panel';
 
 export interface IStandardSkeleton extends StandardProps {
 	className?: string;
-	/** Displays LoadingSkeleton custom header. */
-	header?: React.ReactNode;
```

Comment:
Header has been moved to ILoadingSkeletonProps to allow itself to be set on 'infrastructure' LoadingSkeleton instead of individual skeletons